### PR TITLE
Update phase-3-bootstrap-prompt.md

### PR DESCRIPTION
**Pull request by AmaanRS**
Added real life usage of frameworks

### CODE REVIEW

File: projects/recipe-page/phase-3-bootstrap-prompt.md [hunk 1]
```diff
@@ -158,3 +158,11 @@ Let's clean up the navbar a little bit. You can remove things like the search ba
 ### Extensions
 
 Find some other Bootstrap components you can add to your webpage!
+
+### Real life usage of frameworks
```
Comment (daaimah123) [inline projects/recipe-page/phase-3-bootstrap-prompt.md:162]: @AmaanRS, 

- Please move this new section under the first "NOTE" on the page.
- Please also remove the spaces between each bulleted item, to maintain uniformity amongst bullets across the markdown.
- Please add a bullet about "being able to explain in an interview setting about how they were able to adapt a project"

Comment (AmaanRS) [reply projects/recipe-page/phase-3-bootstrap-prompt.md]: I made the changes.

**Review by daaimah123 — APPROVED** [review]

**Comment by daaimah123** [comment]
@AmaanRS I realize after merging in your PR, that you are not listed as a contributor in our project. Please kindly follow [the contributor steps](https://github.com/Techtonica/curriculum/blob/efc365aba1bf1ea425060369751d2bc048174b22/CONTRIBUTING.md) to have future work added to the repo.

**Comment by daaimah123** [comment]
@AmaanRS I realize after merging in your PR, that you are not listed as a contributor in our project. Please kindly follow [the contributor steps](https://github.com/Techtonica/curriculum/blob/efc365aba1bf1ea425060369751d2bc048174b22/CONTRIBUTING.md) to have future work added to the repo.

**Pull request merged**